modules/govuk/manifests/apps/local_links_manager.pp

```diff
@@ -43,6 +43,12 @@
 #   Sets the OAuth Secret Key for using GDS-SSO
 #   Default: undef
 #
+# [*redis_host*]
+#   Default: undef
+#
+# [*redis_port*]
+#   Default: 6379
```

Comment:
This is a bit nitpicky (sorry) but can we explain here that these parameters set environment variables for a Redis connection?